battery-capacity/interface.js

```diff
@@ -0,0 +1,82 @@
+import { getRoute } from './client';
+import { drawRoutePolyline } from './map';
+import { getDurationString } from '../utils';
+
+// This example uses a Tesla 75D (2016-2019). By default it has a useable battery capacity of 72.5 kWh and
+// a Chargetrip best range of 389. These two values are interchangeable when adjusting the capacity.
+// In a real application you would fetch these values on a per car basis using a car request.
+// See our car example on how you would get these details; https://examples.chargetrip.com/?id=car-list
+const usableBatteryCapacityInKWH = 72.5;
+const chargetripBestRange = 389;
+
+/**
+ * Function that renders the header details
+ * @param { object } data - All available route data
+ */
+export const renderRouteHeader = data => {
```

Comment:
This function needs to be called when the route has been updated.

Reply:
Fixed